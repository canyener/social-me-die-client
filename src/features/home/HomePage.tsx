import React from 'react'
import { Button, Container, Header, Segment, Image } from 'semantic-ui-react'
import { Link } from 'react-router-dom'

const HomePage = () => {
  return (
    <Segment inverted textAlign='center' vertical className='masthead'>
      <Container text>
        <Header as='h1' inverted>
          <Image
            size='massive'
            src='/assets/logo.png'
            alt='logo'
            style={{ marginBottom: 12 }}
          />
          social-me-die
        </Header>
        <Header as='h2' inverted content='Welcome to my little social media experiment' />
        <Button as={Link} to='/activities' size='huge' inverted>
          Take me to the activities!
        </Button>
      </Container>
    </Segment>
  )
}

export default HomePage