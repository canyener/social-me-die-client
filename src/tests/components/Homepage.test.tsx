import { shallow } from 'enzyme'
import HomePage from '../../features/home/HomePage'


test('Should render Homepage correctly', () => {
  const wrapper = shallow(<HomePage />)
  expect(wrapper).toMatchSnapshot()
})