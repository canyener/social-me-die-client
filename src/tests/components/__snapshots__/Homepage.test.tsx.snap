// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Homepage correctly 1`] = `
<Segment
  className="masthead"
  inverted={true}
  textAlign="center"
  vertical={true}
>
  <Container
    text={true}
  >
    <Header
      as="h1"
      inverted={true}
    >
      <Image
        alt="logo"
        as="img"
        size="massive"
        src="/assets/logo.png"
        style={
          Object {
            "marginBottom": 12,
          }
        }
        ui={true}
      />
      social-me-die
    </Header>
    <Header
      as="h2"
      content="Welcome to my little social media experiment"
      inverted={true}
    />
    <Button
      as={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "displayName": "Link",
          "propTypes": Object {
            "innerRef": [Function],
            "onClick": [Function],
            "replace": [Function],
            "target": [Function],
            "to": [Function],
          },
          "render": [Function],
        }
      }
      inverted={true}
      size="huge"
      to="/activities"
    >
      Take me to the activities!
    </Button>
  </Container>
</Segment>
`;
